spec.go

```diff
@@ -286,15 +286,9 @@ func createLibcontainerConfig(cgroupName string, spec *specs.LinuxSpec) (*config
 		return nil, err
 	}
 	config.Cgroups = c
-	if config.Readonlyfs {
-		setReadonly(config)
```

Comment:
it was setting sys as ro but since this is now specified in the config on the sys mount there was no need to do this special case.  it was just legacy code.